src/configuration/configuration.ts

```diff
@@ -52,3 +63,19 @@ export function pinnedWorkflowsRefreshInterval(): number {
 export function getRemoteName(): string {
   return getConfiguration().get<string>(getSettingsKey("remote-name"), "origin");
 }
+
+export function isUseEnterprise(): boolean {
```

Comment:
Can we simplify the name to:
```suggestion
export function useEnterprise(): boolean {
```

Reply:
implemented by e6fa56779d7445e1ccbbce5cbbc3112922c1529c